cloud-functions/functions/src/index.ts

```diff
@@ -440,3 +440,50 @@ export const initiatePasswordRecovery = functions.https.onCall((body) => {
       });
   });
 });
+
+export const getVerificationCode = functions.https.onCall(async () => {
+  const axios = require('axios');
```

Comment:
Let's make these `ES6` style imports to stay consistent with the rest of the file.